update README.md (#28)
evrythng/digital-link.js: README.md

The grammar files corresponding to each version of the standard can be found in
the `grammar` folder.

The grammar files are used by the [`apglib`](https://github.com/ldthomas/apg-js2) library to generate the `grammarObject.js` file.

> then go to the `grammar` folder and run the following command:

Reviewer: ```suggestion
To update the grammar object go to the `grammar` folder and run the following command:
```

```bash
apg -i GS1_Digital_Link_Grammar_<VERSION_NUMBER>.abnf -o ../lib/grammarObject.js
```

This will generate the `grammarObject.js` file in the `lib` folder.

> You will have to apply little modifications to the generated file to make it work.

Reviewer: ```suggestion
You will have to apply some modifications to the generated file to make it work.
```

#### Generated `grammarObject.js` file
```js
// Generated by JavaScript APG, Version [`apg-js2`](https://github.com/ldthomas/apg-js2)
module.exports = function(){
"use strict";
  //```
  // [...]
}
```

#### Updated `grammarObject.js` file
```js
/* eslint-disable */
  
// Generated by JavaScript APG, Version [`apg-js2`](https://github.com/ldthomas/apg-js2)
module.exports = {
  grammarObject: function(){
  "use strict";
    //```
    // [...]
  }
}
```

> Indeed, the generated file is not exporting an object, so you have to wrap it into an object with `grammarObject` field to be able to use it.

Reviewer: ```suggestion
As the generated file is not exporting an object, so you have to wrap it into an object with `grammarObject` field to be able to use it.
```
